Sources/TokamakCore/Environment/EnvironmentKey.swift

```diff
@@ -17,11 +17,19 @@ public protocol EnvironmentKey {
   static var defaultValue: Value { get }
 }
 
-protocol EnvironmentModifier {
+public protocol _EnvironmentModifier {
```

Comment:
Didn't realize we also have `EnvironmentalModifier`. Should we reconsider the naming of this protocol to avoid any confusion? If there's no other good naming option, we should at least have doc comments that clarify the difference between these protocols.

Reply:
I added a doc comment for now. `EnvironmentalModifier` is a protocol available in SwiftUI that lets a user create a new `ViewModifier` using the environment. `_EnvironmentModifier` is a Tokamak protocol that lets a `View` or `ViewModifier` mutate the `EnvironmentValues`. The names are definitely confusing.

Reply:
Should we keep the names as they are then?

Reply:
I’d be fine renaming it, just don’t have an idea atm. `EnvironmentalModifier` needs to stay the same for compatibility though.

Reply:
Yeah, let's keep it as is for now until we come up with a better name...